Resources/Audio/DeltaV/Jukebox/attributions.yml

```diff
@@ -0,0 +1,77 @@
+# sorted alphabetically based on filenames in the folder Resources/Audio/DeltaV/Jukebox
+# keep it ordered or I'll stab you
+
+- files: ["a_different_reality_lagoona_remix.xm-MONO.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "A.D.R (Lagoona rmx) by Andreas Viklund, converted to mono"
+  source: "https://modarchive.org/index.php?request=view_by_moduleid&query=134786"
+
+- files: ["aggravated.it-MONO.ogg"]
+  license: "CC-BY-NC-SA-4.0"
+  copyright: "MEL -Aggravated Assault by melcom, converted to mono"
+  source: "https://modarchive.org/index.php?request=view_by_moduleid&query=176234"
+
+- files: ["autumnal_equinox.xm-MONO.ogg"]
+  license: "CC-BY-NC-4.0"
+  copyright: "Autumnal Equinox by lemonade, converted to mono"
+  source: "https://modarchive.org/index.php?request=view_by_moduleid&query=143993"
+
+- files: ["DOS=HIGH,_UMB.ogg"]
+  license: "Custom"
+  copyright: "DOS=HIGH, UMB by MASTER BOOT RECORD may be used non-commercially in the Delta-V fork of SS14. Converted from FLAC to OGG, then converted to mono."
+  source: "https://masterbootrecord.bandcamp.com/album/c-edit-config-sys"
+
+- files: ["drozerix_-_alone.xm-MONO.ogg"]
+  license: "Custom"
+  copyright: "Alone by Drozerix, converted to mono"
+  source: "https://modarchive.org/index.php?request=view_by_moduleid&query=199968"
+
+- files: ["drozerix_-_leisurely_voice.xm-MONO.ogg"]
+  license: "Custom"
```

Comment:
can we get "custom" defined so it's recorded somewhere what the license is?

Reply:
That’s how it is labeled in the lobby music. idk what to tell you